lib/casmutils/xtal/lattice.cxx

```diff
@@ -27,5 +28,18 @@ bool LatticeEquals_f::operator()(const Lattice& other)
 {
     return ref_lat.column_vector_matrix().isApprox(other.column_vector_matrix(), tol);
 }
+
+void make_niggli(Lattice* lattice_ptr)
```

Comment:
We can leave this here, but for new stuff we write, let's just skip passing pointers and always do the `const` and return a new instance versions unless we truly need an in place modification.